scripts/build-rollup/index.js

```diff
@@ -0,0 +1,30 @@
+const rollup = require('rollup');
+const plugins = require('./plugins');
+const external = require('./externals');
+const { log } = require('../utils');
+
+const entry = 'src/index.js';
+const dest = 'lib/react-dom-lite.cjs.js';
```

Comment:
i don't think tehre is a need for the `cjs.js` ending

Reply:
Sure. But just a reminder this build isn't browser ready. We might need to indicate that.

Reply:
i think this sort of an odd react convention, most npm distubuted packages indicate via the output folder (/dist, /lib, /es, etc), and package.json browser/main/modules entries/

Reply:
I'm fine with anything :) Following existing conventions make it easier for the user I thought